DS/DSAssignment7/BinarySearch.java

```diff
@@ -0,0 +1,41 @@
+public class BinarySearch {
+
+	private boolean flag = false;
+	private int position = 0;
+
+	public int binarySearch(int input[], int start, int end, int key) {
```

Comment:
ACTION: As I have told u earlier also don't over expose methods/fields to the user. Now, here why should the user enter 'start' and 'end' when it is of internal use and also he will be entering same values every time to get things right.